pkg/helpers/helpers_test.go

```diff
@@ -76,7 +77,7 @@ func TestDecodeSnowflakeParameterID(t *testing.T) {
 // TODO: add tests for non object identifiers
 func TestEncodeSnowflakeID(t *testing.T) {
 	testCases := map[string]struct {
-		identifier        sdk.ObjectIdentifier
+		identifier        any
```

Comment:
sdk.ObjectIdentifier can hold both values and pointers to values, but that's a nitpick

Reply:
Done